src/router/index.js

```diff
@@ -152,7 +164,13 @@ const routes = [
       }
       return { name: 'SignIn' };
     },
-    meta: { pageTitle: 'Sign Out' },
+    meta: {
+      pageTitle: {
+        'en-US': 'Sign Out',
+        en: 'Sign Out',
+        es: 'Cerrar Sesión',
+      },
+    },
```

Comment:
See comment above about moving all translation content to the `translations` directory.